data/tribes/initialization/empire/units.lua

```diff
@@ -1183,6 +1214,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_foresters_house",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Forester's House
+            lore = pgettext("empire_building", "‘Vivat, crescat, floreat!’ (Live, grow, and bloom!)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Forester's House
+            lore_author = pgettext("Blessing from Goddes Ceres"),
```

Comment:
Nice, but Ceres is goddess of agriculture, so less fit for forester.

Reply:
```suggestion
            lore_author = pgettext("Blessing from the god Silvanus"),
```
https://en.wikipedia.org/wiki/Silvanus_(mythology)

Reply:
Perfect :)